docs/stable/2-account.md

```diff
@@ -120,7 +111,7 @@ not do this.
 | ---- | ----------- |
 | privkey | ECDSA secp256k1 private key |
 | pubkey | ECDSA secp256k1 public key |
-| whisper key | pubkey for chat with HD derivation path m/44'/60'/0'/0/0 |
+| whisper key | pubkey for chat with HD derivation path m/43'/60'/1581'/0'/0 |
```

Comment:
What is this change?

Reply:
This is me rebasing against `develop` instead of `master`. Fixed.